frontend/src/operator/webui/src/app/safe-device-url.pipe.ts

```diff
@@ -10,7 +10,7 @@ export class SafeDeviceUrlPipe implements PipeTransform {
   transform(deviceId: string) {
     /* eslint-disable */
     // DO NOT FORMAT THIS LINE
-    return this.sanitizer.bypassSecurityTrustResourceUrl(`/devices/${deviceId}/files/client.html`);
+    return this.sanitizer.bypassSecurityTrustResourceUrl(window.location + `/devices/${deviceId}/files/client.html`);
```

Comment:
I guess `this.sanitizer.bypassSecurityTrustResourceUrl(`devices/${deviceId}/files/client.html`);`(omitting /) might work.